.github/workflows/tag.yml

```
@@ -0,0 +1,104 @@
+      ) && (
+        github.event.issue.user.login == 'lukasheinrich' ||
+        github.event.issue.user.login == 'matthewfeickert' ||
+        github.event.issue.user.login == 'kratsg'
+      )
+    env:
+      IS_MAJOR: >-
+          ${{ startsWith( github.event.issue.title, '[bumpversion:major]' ) }}
+      IS_MINOR: >-
+          ${{ startsWith( github.event.issue.title, '[bumpversion:minor]' ) }}
+      IS_PATCH: >-
+          ${{ startsWith( github.event.issue.title, '[bumpversion:patch]' ) }}
+    runs-on: ubuntu-latest
+    steps:
+    - name: determine part
+      run: |
+        if [ $IS_MAJOR == 'true' ]
+        then
+          echo "::set-env name=BV_PART::major"
+        elif [ $IS_MINOR == 'true' ]
+        then
+          echo "::set-env name=BV_PART::minor"
+        else
+          echo "::set-env name=BV_PART::patch"
+        fi
+    - name: add bumpversion/${{ env['BV_PART'] }} label
+      uses: actions/github@v1.0.0
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        args: label bumpversion/${{ env['BV_PART'] }}
+    - uses: actions/checkout@master
+    - name: Prepare repository
+      run: git checkout "${GITHUB_REF:11}"
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: install bumpversion
+      run: |
+        python -m pip install --upgrade pip setuptools wheel
+        pip install bumpversion
+    - name: set up git user
+      run: |
+        git config --local user.email "action@github.com"
+        git config --local user.name "GitHub Action"
+    - name: run bumpversion ${{ env['BV_PART'] }}
+      env:
+        ISSUE_NUMBER: ${{ github.event.issue.number }}
+        ISSUE_DESCRIPTION: ${{ github.event.issue.description }}
+      run: |
+        echo "bumpversion $BV_PART"
+        OLD_TAG=$(git describe --tags --abbrev=0)
+        echo "::set-env name=OLD_TAG::$OLD_TAG"
+        bumpversion $BV_PART --message "Bump version: {current_version} → {new_version}
+
+        Resolves #${ISSUE_NUMBER} via GitHub Actions."
+        NEW_TAG=$(git describe --tags --abbrev=0)
+        echo "::set-env name=NEW_TAG::$NEW_TAG"
+        git tag -n99 -l $NEW_TAG
+
+        CHANGES="$(git log --pretty=format:'%s' $OLD_TAG..HEAD -i -E --grep='^([a-z]*?):' | sed 's/^/  - /')"
+        NUM_CHANGES=$(echo -n "$CHANGES" | grep -c '^')
+        echo "::set-env name=NUM_CHANGES::$NUM_CHANGES"
+        git tag $NEW_TAG $NEW_TAG^{} -f -m "$(printf "This is a $BV_PART release from $OLD_TAG → $NEW_TAG.\n\nChanges:\n$CHANGES")"
+        git tag -n99 -l $NEW_TAG
+    - name: comment on issue
+      uses: actions/github@v1.0.0
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        args: |
+          comment "This is a ${{ env['BV_PART'] }} release from ${{ env['OLD_TAG'] }} → [${{ env['NUM_CHANGES'] }} change(s)] → ${{ env['NEW_TAG'] }}. I'm going to attempt to push now."
```

```suggestion
          comment "This is a ${{ env['BV_PART'] }} release from ${{ env['OLD_TAG'] }} → ${{ env['NEW_TAG'] }} with ${{ env['NUM_CHANGES'] }} change(s). I'm going to attempt to push now."
```

I'd suggest having the number of changes not be part of the tag change chain. As we only ever want releases to be _just_ releases, then can we have it succeed only if the number of changes is 1?